rancher/dashboard
# Branding settings: Login Background & Banner

### Summary
Fixes https://github.com/rancher/dashboard/issues/10423
Bocked by: https://github.com/rancher/rancher/issues/44577


### Occurred changes and/or fixed issues


We add new settings to customize the Login Background and Dashboard's Homepage banner.

### Technical notes summary


New settings required:

1. `ui-login-background-light` - for the background, light theme image
2. `ui-login-background-dark` - for the background, dark theme image
3. `ui-banner-light` - for the banner, light theme image
4. `ui-banner-dark` - for the banner, dark theme image

### How to test

- Run Rancher server locally, using this [branch](https://github.com/rancher/rancher/pull/44961)
  - https://github.com/rancher/rancher/wiki/Setting-Up-Rancher-2.0-Development-Environment#set-up-vscode
  or
  - See internal documentation

- Run a Rancher UI locally - `API=https://127.0.0.1:8443 yarn dev`

### Areas or cases that should be tested




- General Settings -> Branding page
- Login Background
- Home Page -> banner on top

### Areas which could experience regressions


- Settings -> Branding page

### Screenshot/Video


Banner in Homepage

![image](https://github.com/rancher/dashboard/assets/26394656/bc438793-88e3-46c1-b01a-b646e8e405f8)

Branding tab

![image](https://github.com/rancher/dashboard/assets/26394656/6449065d-b96a-4402-b5bd-6d11e7fcd9e0)


### Checklist
